content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -160,6 +160,26 @@ try {
 {{% /tab %}}
 {{< /tabs >}}
 
+## Troubleshooting
+
+### Fix "Script error."
+
+{{< img src="real_user_monitoring/browser/script-error.png" alt="Real User Monitoring script error example" style="width:75%;" >}}
+
+For security reasons, browsers hide details from errors triggered by cross-origin scripts ([see CORS][6] for more information). Some of the reasons why you are seeing this error are:
```

Comment:
For more information, [MDN explanation about the topic](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror#notes) could be better than redirecting to CORS doc